app/tests/cases_tests/test_widget.py

```diff
@@ -204,26 +204,42 @@ def test_flexible_image_widget_prepopulated_value():
     ci = ComponentInterfaceFactory(kind=ComponentInterface.Kind.PANIMG_IMAGE)
     civ = ComponentInterfaceValueFactory(interface=ci, image=im)
 
-    field = InterfaceFormFieldFactory(
-        interface=ci, user=user_with_perm, initial=civ
+    field = (
+        InterfaceFormFieldsMixin()
+        .get_fields_for_interface(
+            interface=ci, user=user_with_perm, initial=civ
+        )
+        .popitem()[1]

```

Comment:
This implicitly adds an ordering to the test, while we don't really care about this. I think. Plus it's pretty hard to read.

What about this instead?

```Python
fields = InterfaceFormFieldsMixin().get_fields_for_interface(
     interface=ci, user=user_with_perm, initial=civ
)
assert len(fields) == 1
field = fields[f"{INTERFACE_FORM_FIELD_PREFIX}{ci.slug}"]
```